common-authentication/src/main/java/org/alfresco/hxi_connector/common/adapters/auth/AuthenticationResult.java

```diff
@@ -47,16 +47,13 @@
 @AllArgsConstructor
 public class AuthenticationResult
 {
-    @NotBlank
-    @JsonProperty("access_token")
+    @NotBlank @JsonProperty("access_token")
     String accessToken;
     @Positive @JsonProperty("expires_in")
     int expiresIn;
     TemporalUnit temporalUnit = ChronoUnit.SECONDS;
-    @NotBlank
-    @JsonProperty("token_type")
+    @NotBlank @JsonProperty("token_type")
     String tokenType;
-    @NotBlank
-    String scope;
+    @NotBlank String scope;
```

Comment:
Are you using the spotless formatter? I'm surprised that it's changed this file. In fact I don't think there are any code changes here so please reset the copyright header, revert the formatting and commit with `--no-verify`.